.github/workflows/test.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         run: dart pub get
 
       - name: Format
-        run: dart format --set-exit-if-changed -l 160 ./lib
+        run: dart format --set-exit-if-changed -l120 ./lib
```

Comment:
Why?

Reply:
because main nyxx package always had 120 https://github.com/nyxx-discord/nyxx/blob/main/Makefile#L5 . Making formatting consistent

Reply:
https://github.com/nyxx-discord/nyxx/blob/5d60dcc35b26edadfa22f7919df74ece448f85ce/.github/workflows/test.yml#L81

Why it's 160 in the actions and in CONTRIBUTING.md then?

Reply:
uuu, then I got myself confused. Fixing